src/tools/pythongen/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-return ()
+remove_tool (pythongen "Deprecated and will be removed soon")
```

Comment:
Maybe it is better if you directly removed it if this PR breaks it.

Reply:
Yes, I'll do that. I just wanted to get the PR in a state where all CI jobs run successfully. Small incremental changes from a working state are much easier.

Reply:
> Also, are we going to remove the old pythongen stuff for 1.0? Because then I won't bother fixing the remaining bugs...

Yes, let us remove it. Later we can port the code generator for contextual values to your new template system. Its low-priority for 1.0, though.